anaconda_project/project_commands.py

```diff
@@ -408,6 +408,8 @@ def description(self):
             description = self._attributes.get('windows', None)
         if description is None:
             description = self._attributes.get('conda_app_entry', None)
+        if description is None:
+            description = getattr(self, 'command', None)
```

Comment:
I don't mind the name it was just confusing to me that there's no trace of this attr in the base class. Maybe set it to None in the base? Or make it an instance attr None by default?